### PR TITLE
Update bootstrap to v5

### PR DESCRIPTION
**Pull request by pakb**
This update is there in order to avoid the Sass hell we will (soon) face if we do not do that. The newer interpreter that is coming will not allow calculations such as `myVar: $variable / 2` anymore (but only want multiplications => `myVar: $variable * 0.5`)
The version 5 of bootstrap has this changes embedded (whereas v4 doesn't yet).
I've also updated sass and sass-loader accordingly

I don't really know why but then the linter wants to lint a bit more the code after this update.
There's a specific commit for linting, if that helps reviewing this PR...

[Test link](https://web-mapviewer.dev.bgdi.ch/update_bootstrap/index.html)

### CODE REVIEW

**Comment by pakb** [comment]
I still need to fix drawing style popup (marker and such) with the newer version of `popper.js`

File: src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue [hunk 1]
```diff
@@ -43,37 +65,14 @@ export default {
...
-            html: true,
-        })
-        this.popover.on('inserted.bs.popover', () => {
-            jquery('.timestamps-popover-container').append(jquery(this.timestampSelectionList))
-            jquery('.btn-timestamp').on('click', (event) => {
-                this.$emit('timestampChange', jquery(event.target).attr('data-timestamp'))
+        if (this.hasMultipleTimestamps) {
+            this.popover = Popover.getOrCreateInstance(this.$refs.popoverButton, {
+                content: this.$refs.popoverContent,
+                html: true,
```
Comment (boecklic) [inline src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue:73]: nice!

File: src/utils/coordinateUtils.js [hunk 2]
```diff
@@ -194,29 +198,22 @@ const executeAndReturn = (
  * E.G. `'47.1, 7.5'` is valid and will return `[47.1, 7.5]` but `'lat:47.1, lon:7.5'` will fail and
  * return `undefined`.
  *
- * Separators
- *     ------------------------------------------------
- * To separates the two numerical values, a combination of slashes, spaces (tabs included) or a coma
- * can be used.
+ * Separators ------------------------------------------------ To separates the two numerical
+ * values, a combination of slashes, spaces (tabs included) or a coma can be used.
  *
```
Comment (boecklic) [inline src/utils/coordinateUtils.js:203]: this probably linter gone wild?

Comment (boecklic) [inline src/utils/coordinateUtils.js:203]: this probably linter gone wild?

File: src/utils/militaryGridProjection.js [hunk 3]
```diff
@@ -39,8 +39,7 @@ const Z = 90 // Z
  *
  * @param {[number, number]} ll Array with longitude and latitude on a WGS84 ellipsoid.
  * @param {number} [accuracy=5] Accuracy in digits (5 for 1 m, 4 for 10 m, 3 for 100 m, 2 for 1 km,
- *   1 for 10 km or 0 for 100 km). Optional, default is
- *   5. Default is `5`
+ *   1 for 10 km or 0 for 100 km). Optional, default is 5. Default is `5`
```
Comment (boecklic) [inline src/utils/militaryGridProjection.js:42]: duplicate 'default is 5'

**Review by boecklic — APPROVED** [review]
:+1:

**Comment by pakb** [comment]
Replaced by https://github.com/geoadmin/web-mapviewer/pull/94 so that I can split my work (bootstrap and drawing review/small fix) into two PRs

**Pull request closed without merging**